src/main/java/carracing/model/RandomStrategy.java

```diff
@@ -0,0 +1,12 @@
+package carracing.model;
+
+import java.util.Random;
+
+public class RandomStrategy implements MovingStrategy {
+    private static final Random random = new Random();
```

Comment:
정적 상수 이니 대문자로 수정되어야 할 것 같습니다 🙇 
```suggestion
    private static final Random RANDOM = new Random();
```